src/redux/Tasks/reducers.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Task related reducers
+ */
+import { omit } from 'lodash'
+import moment from 'moment/min/moment-with-locales'
+import {
+  LOAD_TASKS_REQUEST, LOAD_TASKS_FAILURE, LOAD_TASKS_SUCCESS,
+  MARK_TASK_DONE_REQUEST, MARK_TASK_DONE_FAILURE, MARK_TASK_DONE_SUCCESS,
+  MARK_TASK_FAILED_REQUEST, MARK_TASK_FAILED_FAILURE, MARK_TASK_FAILED_SUCCESS,
+  DONT_TRIGGER_TASKS_NOTIFICATION,
+} from './actions'
+import { MESSAGE } from '../middlewares/WebSocketMiddleware'
+import { insertAt } from '../util'
+import { localeDetector } from '../../i18n'
+
+
+moment.locale(localeDetector())
+
+
+/*
+ * Intital state shape for the task entity reducer
+ */
+const tasksEntityInitialState = {
+  fetchError: null,                // Error object describing the error
+  isFetching: false,               // Flag indicating active HTTP request
+  isStale: false,                  // Flag indicating state fragment should be re-fetched
```

Comment:
Nice

Reply:
Yes, it's not used yet. I couldn't quite figure out what the best condition would be under which to mark tasks as stale. Maybe I should just remove it, it's easy enough to add it back if we decide we need it.

What do you think?

Reply:
I am not sure you need this boolean, as you can check `fetchError === null` (or sthg equivalent) to know if you failed to fetch.
In any case please remove it if it is not used.